locales/templates/app.pot

```diff
@@ -531,15 +531,27 @@ msgctxt "firstrun_next_button"
 msgid "Next"
 msgstr ""
 
-#. The welcome titles in Firefox Lite 2.5
+#. !NOT USE! The welcome titles in Firefox Lite 2.5
 msgctxt "firstrun_fxlite_2.5_title_1"
 msgid "Hello!"
 msgstr ""
 
+#. !NOT USE! The welcome titles in Firefox Lite 2.5 for asking users to choose
+#. a main purpose of downloading Firefox Lite. The %1$s is Firefox Lite.
+#, c-format
+msgctxt "firstrun_fxlite_2.5_title_2"
+msgid "What brings you to %1$s?"
+msgstr ""
+
+#. The welcome titles in Firefox Lite 2.5
+msgctxt "firstrun_fxlite_2_5_title_A"
+msgid "Hello!"
+msgstr ""
+
 #. The welcome titles in Firefox Lite 2.5 for asking users to choose a main
 #. purpose of downloading Firefox Lite. The %1$s is Firefox Lite.
 #, c-format
-msgctxt "firstrun_fxlite_2.5_title_2"
+msgctxt "firstrun_fxlite_2_5_title_B"
```

Comment:
Is there a reason to change this although the string remains the same?

Reply:
We try to not using "." in the string ID because our developers report some bugs when there're "." in the string ID. Even if the Android system automatically replaces the "2.5" by "2_5". So the current walk around solution is to rename the string ID so that there will be no ID with "." in Firefox Lite.

Reply:
Ah I see. Ok it's fine to change then, just wanted to check the rational behind this